enhancements/machine-config/manage-boot-images.md

```diff
@@ -0,0 +1,264 @@
+---
+title: manage-boot-images
+authors:
+  - "@djoshy"
+reviewers: 
+  - "@yuqi-zhang"
+  - "@mrunal"
+  - "@cgwalters, for rhcos context" 
+  - "@joelspeed, for machine-api context" 
+  - "@sdodson, for installer context"
+approvers:
+  - "@yuqi-zhang"
+api-approvers: 
+  - None
```

Comment:
This enhancement has an API element to it, assign me as the API approver

Reply:
Missed that, will do!